src/components/dashboard/FeedList.js

```diff
@@ -228,6 +231,7 @@ const FeedList = ({
         onMomentumScrollEnd={_onScrollEnd}
         refreshing={false}
         renderItem={renderItemComponent}
+        ListHeaderComponent={renderHeaderComponent}
```

Comment:
```suggestion
        ListHeaderComponent={listHeaderComponent}
```

could be a jsx element https://reactnative.dev/docs/flatlist#listheadercomponent